support-frontend/assets/pages/[countryGroupId]/components/legend.tsx

```diff
@@ -0,0 +1,23 @@
+import { css } from '@emotion/react';
+import { from, headlineBold24, space } from '@guardian/source/foundations';
+import type { ReactNode } from 'react';
+
+const legend = css`
+	margin-bottom: ${space[3]}px;
+	${headlineBold24};
+	${from.tablet} {
+		font-size: 28px;
+	}
+
+	display: flex;
+	width: 100%;
+	justify-content: space-between;
+`;
+
+type LegendProps = {
+	children: ReactNode;
+};
+
+export function Legend({ children }: LegendProps) {
+	return <legend css={legend}>{children}</legend>;
+}
```

Comment:
As per the above - a `Legend` would never live outside of a `form`, so I wonder if it's worth containing these styles together.

Reply:
Ideally I would have had one component that took the legend children and the rest of the fieldset's children as props. But alas we re-use the legend styling within a differently styled fieldset